src/style.css

```
@@ -1 +1,64 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav__wraper {
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  font-size: 12px;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  color: #000;
+  height: 60px;
+}
+
+a:hover,
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
```

no need to add position relative when an element is hovered

```suggestion
.nav__link {
  font-size: 12px;
  text-transform: uppercase;
  display: flex;
  align-items: center;
  color: #000;
  height: 60px;
}

nav__link:hover {
  color: #00acdc;
}
  
.is-active {
  position: relative;
  color: #00acdc;
}
```